chaosinstana/api.py

```diff
@@ -44,6 +44,10 @@ def get_all_events(from_time: str, to_time: str,
                 secrets.get("instana_api_token"))
         }
     )
+    if r.status_code > 399:
+        raise ActivityFailed("failed to call '{u}': {c} => {s}".format(
+            u=url, c=r.status_code, s=r.text))
+
     logger.info("Instana reponse code: {}".format(r.status_code))
```

Comment:
Should this remain a logger.info here?

Reply:
Should be logger.debug to avoid pollution?

Reply:
I think so in this case.

Reply:
amended logging to debug, also pulled common code block into new function to wrap get requests